core/llm_client.py

```
@@ -137,8 +137,11 @@ async def generate_img(self, prompt) -> Image:
         logger.info(f"Generating image using {model_id} ({provider_name})")
         try:
             img_res = await image_model.text_to_image(prompt)
-            if not img_res:
-                logger.error(f"Failed to generate image with text")
+            if img_res:
+                logger.info(f"Image generated with prompt: {prompt}")
+                logger.debug(f"type={img_res.image_type}, len={len(img_res.image or '')}, prefix={repr((img_res.image or '')[:200])}")
```

Ruff (0.15.12)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)